DasharoPayloadPkg/Library/FmpDeviceSmmLib/FmpDeviceSmmLib.c

```diff
@@ -1009,6 +1009,17 @@ FmpDeviceSetImageWithStatus (
       continue;
     }
 
+    // Only touch blocks that are fully unlocked for writing.
+    if (!IsRangeWriteable(CurrentImage, BlockSize * BlockCount, Offset, BlockSize)) {
+      DEBUG ((
+        DEBUG_ERROR,

```

Comment:
```suggestion
        DEBUG_INFO,
```
Since we expect this to happen and keep going assuming the update will succeed, I think this should be `INFO`.

Reply:
https://github.com/Dasharo/edk2/pull/284/commits/946cbfb67abd413031a270869c3cb36de28babd7